google/cloud/security/scanner/scanners/iam_rules_scanner.py

```diff
@@ -46,8 +54,160 @@ def __init__(self, global_configs, snapshot_timestamp):
         """
         super(IamPolicyScanner, self).__init__(
             global_configs,
-            snapshot_timestamp)
-        self.snapshot_timestamp = snapshot_timestamp
+            scanner_configs,
+            snapshot_timestamp,
+            rules)
+        self.rules_engine = iam_rules_engine.IamRulesEngine(
+            rules_file_path=self.rules,
+            snapshot_timestamp=self.snapshot_timestamp)
+        self.rules_engine.build_rule_book(self.global_configs)
+
+    def _get_output_filename(self, now_utc):
```

Comment:
Can we move this in the base class? Since I can see use for this in all scanners